src/components/templates/ImportResourceTemplate.jsx

```diff
@@ -12,16 +12,10 @@ import CreateResourceMessages from './CreateResourceMessages'
 import { createResourceTemplate, updateResourceTemplate } from 'sinopiaServer'
 import { getCurrentUser } from 'authSelectors'
 import { clearModalMessages, addModalMessage, showModal } from 'actions/modals'
+import { clearTemplateMessages, setTemplateMessages } from 'actions/flash'
```

Comment:
So there are other sorts of template messages (e.g., error messages). Would be clearer if consistently called these create template messages, like "clearCreateTemplateMessages".

Reply:
These are sometimes "update" messages too.